src/conditions/check_fields.rs

```diff
@@ -17,6 +18,17 @@ pub enum CheckFieldsPredicateArg {
     Boolean(bool),
 }
 
```

Comment:
Hoping someone can point me to a way of avoiding this. All I want here is to format `'foo'` instead of `String(foo)`.

Reply:
I think this is actually relatively clean... :woman_shrugging:

Reply:
Problem with debug formatting is that it includes the enum variant so you get `String(foo)`, `Boolean(false)`, etc. Rather than implementing `fmt` this second way uses https://mcarton.github.io/rust-derivative/Debug.html#hiding-newtypes. Looks a bit ugly though so I'm not sure which I prefer.